loader/src/index.js

```diff
@@ -0,0 +1,31 @@
+import fs from 'mz/fs';
+import path from 'path';
+import {Account, BpfLoader, Connection, PublicKey, testnetChannelEndpoint} from '@solana/web3.js';
+
+(async function() {
+  const url = process.env.LIVE
+  ? testnetChannelEndpoint(process.env.CHANNEL || 'beta', false)
+  : 'http://localhost:8899';
+
+  const distPath = path.join(
+    __dirname,
+    '..',
+    '..',
+    'dist',
+  );
+
+  const elfFile = path.join(distPath, 'programs', 'bandwidth_prepay.so');
+  console.log(`Reading ${elfFile}...`);
+  const elfData = await fs.readFile(elfFile);
+
+  console.log('Loading program...');
+  const loaderAccount = new Account();
+  const connection = new Connection(url);
+  await connection.requestAirdrop(loaderAccount.publicKey, 100000);
+  await BpfLoader.load(connection, loaderAccount, elfData);
+
+  const idFile = path.join(distPath, 'program_id.json');
+  const programId = loaderAccount.publicKey.toBase58();
```

Comment:
@jstarry , the `initialize_contract` issue I mentioned is caused by a bug here. The loaderAccount is the system account used to create and initialize the actual executable.
Delete this line, and see suggestions above and below for fix.
(This now exposes a `TransactionError(InstructionError(0, CustomError(1)))`, which is ResultWithNegativeLamports, and almost certainly related to fees. I'll PR a fix to your branch.)

Reply:
That would do it 🤦‍♂